packages/schema/package.json

```
@@ -34,9 +34,9 @@
 		}
 	},
 	"main": "dist/index.js",
-	"types": "dist/index.d.ts",
 	"files": [
-		"dist"
+		"dist",
+		"!**/*.test.{js,d.ts}"
```

No tests in this package 👍 

```suggestion
		"dist"
```

---

It looked like a package where it would be reasonable for tests to be added in the future (as opposed to `constants` and `types`) so I thought I might as well include it to account for future tests. :) Up to you!